services/datamanager/datasync/sync.go

```diff
@@ -183,12 +183,13 @@ func (s *syncer) syncArbitraryFile(f *os.File) {
 	uploadErr := exponentialRetry(
 		s.cancelCtx,
 		func(ctx context.Context) error {
-			return uploadArbitraryFile(ctx, s.client, f, s.partID)
+			err := uploadArbitraryFile(ctx, s.client, f, s.partID)
+			s.logger.Errorw(fmt.Sprintf("error uploading file %s", f.Name()), "error", err)
+			return err
 		},
-		s.logger,
 	)
 	if uploadErr != nil {
-		s.logger.Error(uploadErr)
+		s.logger.Errorw(fmt.Sprintf("error uploading file %s", f.Name()), "error", uploadErr)
```

Comment:
Believe you should remove here and above now to avoid double logging, since it's already logging this within the exponential retry fn.

Reply:
Removed